client/state/selectors/is-upwork-stats-nudge-dismissed.js

```diff
@@ -0,0 +1,50 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { last } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getPreference } from 'state/preferences/selectors';
+
+const WEEK_IN_MS = 7 * 24 * 60 * 60 * 1000;
+
+/**
+ * Returns the last time the nudge was dismissed by the current user or 0 if it was never dismissed
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}  siteId The Id of the site
+ * @return {Number}  Timestamp marking the last time the nudge was dismissed
+ */
+const getLastDismissTime = ( state, siteId ) => {
+	const preference = getPreference( state, 'upwork-dismissible-nudge' ) || {};
+	const sitePreference = preference[ siteId ] || [];
+	const lastEvent = last( sitePreference.filter( event => 'dismiss' === event.type ) );
+
+	return lastEvent ? lastEvent.dismissedAt : 0;
```

Comment:
That may just be personal preference but I would return `null` instead of `0` here. That way it's very clear that the nudge has never been dismissed - a timestamp of zero is a valid value that indeed corresponds to _Thursday, 1 January 1970_.

Reply:
Going to leave as is for continuity with Google my Business.